content/tutorials/single-page-tutorials/proof-inclusion-of-a-block.md

```diff
@@ -0,0 +1,268 @@
+# Tokenless Data Notarization
+
+This tutorial shows how you can prove that a block was part of the Tangle, even if this part of the Tangle was already pruned from network nodes.
+
+## User Story
+
+A **Prover** wants to prove to a **Verifier** that a dataset or file was not altered by notarizing it using the public and permissionless Tangle. 
+
+A good example could be the government (**Verifier**) obligating companies (**Prover**) to write their daily CO2 emissions to the Tangle to create immutable logs. The government has no interest in providing storage for all the companies but wants to receive verifiable, immutable data in the case of an audit. Therefore, all the data remains with the respective company until an audit occurs.
+
+### Architecture
+
+In this use case, the party writing to the Tangle (**Prover**) is the same party reading from it. This allows reading a block by its block Id together with proof. All a **Verifier** needs to verify this proof is the chain of milestones back to the milestone that referenced this exact block. Every milestone directly references the previous milestone, which leads to a unique and verifiable chain of milestones in the Tangle. Even if the **Verifier's** notarization plugin didn't record the milestone history already, it could still download the milestone chain from another source and verify its correctness before using it to notarize verifications.
+
+Since the pattern assumes trusted access to the network, we recommend that both the **Prover** and the **Verifier** operate their own network node and run the [INX Notarization/PoI Plugin](https://github.com/iotaledger/inx-poi). It is also noteworthy that this pattern does not rely on a Permanode storing the entire Tangle history, drastically reducing the node operator's storage requirements.
```

Comment:
```suggestion
Since the pattern assumes trusted access to the network, we recommend that both the **Prover** and the **Verifier** operate their own network node and run the [INX Notarization/PoI Plugin](https://github.com/iotaledger/inx-poi). As this pattern does not rely on a Permanode storing the entire Tangle history, it consumes much less storage space.
```